test/test-project/test-registry.test.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * @jest-environment node
+ */
+
+const path = require('path');
+const Generator = require('@asyncapi/generator');
+const templateName = '@asyncapi/html-template';
+const tempOutputResults = '../temp/integrationTestResult';
+
+console.log = jest.fn();
+
+describe('Testing if html template can fetch from private repository', () => {
+  jest.setTimeout(200000);
+
+  let ArboristMock;
+  let arboristMock;
+
+  beforeEach(() => {
+    ArboristMock = require('@npmcli/arborist');
+    arboristMock = new ArboristMock();
+  });
```

Comment:
in case of this test, I guess we do not want to mock arborist but actually perform a real installation of the template from private registry

Reply:
@derberg removed